loader/cosmwasm.go

```diff
@@ -0,0 +1,7 @@
+package loader
+
+// mergeCosmwasmPaths overrides packages that need to be modified in order to support
```

Comment:
🤯 powerful but a bit scary as well

Reply:
It would be nice to have this under build flags, so a no op for other targets...
I mean, I try to keep not break the other upstream functionality, so one day we might get this in upstream